spec/unit/puppet_webhook_spec.rb

```diff
@@ -6,4 +6,12 @@
     expect(last_response).to be_ok
     expect(last_response.body).to eq('{"status":"success","message":"running"}')
   end
+
+  it '/echo returns parsed payload' do
```

Comment:
Testing the parser via a route might be easier than testing the parser class directly?